### PR TITLE
Fixed loadValidations for Windows

### PR DESCRIPTION
**Pull request by kouts**

### CODE REVIEW

File: src/validations.js [hunk 1]
```diff
@@ -18,8 +19,9 @@ export function useValidationsStore() {
 export async function loadValidations(keyword) {
   const store = useValidationsStore()
   if (!store.loaded) {
-    const file = await getFilesWithKeyword(keyword)
-    const res = await fetch('../../../' + file[0].vUrl)
+    const files = await getFilesWithKeyword(keyword)
+    const path = getPitcherFolderPath() + files[0].vUrl
```
Comment (digi604) [inline src/validations.js:23]: would create a function that takes a vURL and returns a path instead of this

Comment (kouts) [reply src/validations.js:23]: Ok, like `getFullUrl(vUrl)` ?

Comment (digi604) [reply src/validations.js:23]: yeah or getFullPath or something

Comment (kouts) [reply src/validations.js]: Fixed with 3eaab2b and 1d2d062

**Review by digi604 — APPROVED** [review]

**Pull request merged**